<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>151</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>151</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>151</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0.011, stop:0 rgba(0, 0, 127, 255), stop:1 rgba(94, 122, 255, 255))
}

QLineEdit{
	border-radius: 10px;
	min-height: 25px;
}

QLabel{
	color: white;
	font-weight: bold;
}

QPushButton{
	border: none;
}

QPushButton#start_button{
	border: 2px solid white;
	border-radius: 10px;
	min-height: 25px;
	background-color: white;
	font-weight: bold;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="ui_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Ui Files location</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="ui_edit"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="ui_open_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>open-folder.png</normaloff>open-folder.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="python_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Python Files location</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="python_edit"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="python_open_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>open-folder.png</normaloff>open-folder.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="start_button">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
